sys/netinet/ip_carp.c

```diff
@@ -2091,6 +2117,22 @@ carp_sc_state(struct carp_softc *sc)
 	}
 }
 
+static int
+carp_allow_sysctl(SYSCTL_HANDLER_ARGS)
+{
+	int new, error;
+
+	new = V_carp_allow;
+	error = sysctl_handle_int(oidp, &new, 0, req);
+	if (error || !req->newptr)
+		return (error);
+
+	atomic_swap_int(&V_carp_allow, new);
```

Comment:
I think this atomic operation makes no sense.  It is equivalent to just V_carp_allow = new.  If the output value is ignored, this function is equivalent to atomic_store_int(), which in its place pointless when used without "rel" suffix, since int assignments assumed to be atomic on all platforms by definition.  atomic_store_rel_int() could possibly have some sense.